modules/data.atmosphere/R/met.process.R

```diff
@@ -93,7 +99,7 @@ met.process <- function(site, input_met, start_date, end_date, model, host, dbpa
     }else{
 
       args <- list(outfolder, start_date, end_date)
-      if(met %in% "CRUNCEP") {
+      if((met %in% "CRUNCEP") | (met %in% "GFDL")) {
```

Comment:
Agree. This was a quick throw in. James is doing more edits to make it actually work, have a set of defaults for GFDL, and eventually generalize this by tag.